_posts/news/2015-08-31-announcing-scalajs-0.6.5.md

```diff
@@ -0,0 +1,232 @@
+
+We are thrilled to announce the release of Scala.js 0.6.5!
+
+This is probably *the most important release since Scala.js 0.5.0*!
+It brings significant advances to interoperability with JavaScript:
+
+* You can now implement in Scala.js subclasses of JavaScript classes and traits!
+* Using the same semantics, you can now define object literals with `new js.Object { val x = 5 }`
+* There is a new unboxed pseudo-union type `A | B` to type more accurately your JavaScript facades
+* You can statically typecheck that the `@JSExport`s of a Scala class comply with a JavaScript facade trait with `js.use(x).as[T]`
+
+## Getting started
+
+If you are new to Scala.js, head over to
+[the tutorial]({{ BASE_PATH }}/doc/tutorial.html).
+
+## Release notes
+
+For changes in the 0.6.x series compared to 0.5.x, read [the announcement of 0.6.0]({{ BASE_PATH }}/news/2015/02/05/announcing-scalajs-0.6.0/).
+
+As a minor release, 0.6.5 is backward source and binary compatible with previous releases in the 0.6.x series.
+Libraries compiled with earlier versions can be used with 0.6.5 without change.
+However, it is not forward compatible: libraries compiled with 0.6.5 cannot be used by projects using 0.6.{0-4}.
+
+Please report any issues [on GitHub](https://github.com/scala-js/scala-js/issues).
+
+## Important new warnings
+
+When upgrading from 0.6.{0-4}, you will get new warnings on the declarations of your facade types, i.e., traits, classes and objects extending `js.Any`.
+For example:
+
+{% highlight scala %}
+import scala.scalajs.js
+
+class Foo extends js.Object {
+  def bar(x: Int): Int = js.native
+}
+{% endhighlight %}
+
+will have the following warning:
+
+    Foo.scala:3: warning: Classes, traits and objects inheriting from js.Any should be annotated
+      with @js.native, unless they have @ScalaJSDefined.
+      The default will switch to Scala.js-defined in the next major version of Scala.js.
+    class Foo extends js.Object
+          ^
+
+As the text says, you should simply add the `@js.native` annotation to the declaration of `Foo` to silence the warning:
+
+{% highlight scala %}
+import scala.scalajs.js
+
+@js.native
+class Foo extends js.Object {
+  def bar(x: Int): Int = js.native
+}
+{% endhighlight %}
+
+Addressing these warnings is important to make your source code forward compatible with the next major version of Scala.js.
+An unannotated declaration extending `js.Any` will by default be Scala.js-defined (see next section) in the next version.
+
+## Improvements
+
+### Scala.js-defined JS classes, objects, and traits
+
+Scala.js 0.6.5 introduces a major language improvement: the ability to define, in Scala.js, a subclass of a native JavaScript class (or implementing a JavaScript trait/interface).
+We call such classes *Scala.js-defined JS classes*, because they are effectively JavaScript classes, but written in Scala.js.
+This is opposed to the classes that you typically write, which are Scala classes, not JavaScript classes.
```

Comment:
I would not use opposed unless you have a second, dependent phrase. SJS defined classes are not opposed to Scala classes. They complement :)